packages/stencil-components/stencil.config.ts

```diff
@@ -0,0 +1,26 @@
+import { Config } from '@stencil/core';
+import { sass } from '@stencil/sass';
+import { resolve, dirname } from 'path';
+
+const ROOT_DIR = resolve(__dirname, '../../');
+const ODYSSEY_DIR = dirname(require.resolve('@okta/odyssey/package.json'));
+
+export const config: Config = {
+  namespace: 'o-components',
```

Comment:
since these are odyssey components, I would strongly prefer we use the `ods` prefix as much as possible for consistency.

Reply:
Glad you commented on this! Was thinking the same thing.

Today, our web components utilize the `o-${type}` convention, so there would be a significant refactor required to update our markup and testing snapshots, to support a new prefix. While a straightforward change, it's a costly one. We could choose to rename this now, but I'd argue can be put off until later.

I'd consider these our priorities in sequential order:
1. Migrate components from `okta-ui` to `odyssey`
2. Utilize the newly published Odyssey components in Okta's client apps
   - _Should_ be a simple dependency change
   - **Important**: There should be no changes to markup at this time.
3. Enhance our components via API, markup, or name changes.

Reply:
This feels problematic to me. While significant refactors may be needed, they won't become easier to do if we're gated by in-use UI.

If we follow this priority order, then components that aren't written to Odyssey spec - even if it's something as small as a prefix - become Odyssey official and change-risky because they're in use today.

To @jeffbaltazar-okta's earlier concern, since nearly none of the UI Core code, UI, or behavior been reviewed by the Odyssey team, we become locked in at a starting point we haven't reached alignment on. 

Because of (2), we may find ourselves stuck there, waiting for 👍 from a product team to make those changes.

e.g. The prefixes in this self-same PR.